src/utils/helpers.ts

```diff
@@ -92,3 +93,8 @@ export const deepMap = (transform: (arg: any) => any, object: DeepMappable): any
 
 export const cveRegex = /^CVE-(?:1999|2\d{3})-(?!0{4})(?:0\d{3}|[1-9]\d{3,})$/;
 export const uniques = <T>(array: T[]) => Array.from(new Set(array));
+
+export function formatDate(date: Date | string): string {
+  const jsDate = new Date(date); // Handles strings in ISO/component format, and Date object
+  return DateTime.fromJSDate(jsDate, { zone: 'utc' }).toFormat('yyyy-MM-dd T ZZZZ');
+}
```

Comment:
We have almost this exact function in `EditableDate` widget, maybe remove it from the widget and use this helper https://github.com/RedHatProductSecurity/osim/blob/9f06e2dd03988f1236bca8288cfa1ff04e1d01c2/src/components/widgets/EditableDate.vue#L80-L83

Reply:
+1, if @superbuggy doesn't mind to update it.

Reply:
I refactored the helper function so that it's reuseable in `EditableDate`. It seems to be working as imtended.